resources/assets/sass/components/common/static/_faq.scss

```diff
@@ -1,78 +1,78 @@
-.faq {
+// .faq {
```

Comment:
Can we remove this instead of commenting it out? Or keep this styling live and just comment out the import?

Reply:
Sure, find by me. I comment out out of habit, Github retains the memory for me.